src/store/login/useAuthStore.js

```
@@ -70,12 +86,38 @@ const useAuthStore = create((set) => ({
       return false
     }
+    const decoded = parseJwt(token)
+    if (decoded && decoded.exp * 1000 < Date.now()) {
+      // 토큰 만료 시 갱신 시도
+      try {
+        const newAccessToken = await refreshAccessToken(token)
+        const newDecoded = parseJwt(newAccessToken)
+        set({ user: newDecoded, isAuthenticated: true, accessToken: newAccessToken })
+        console.log('새로운 accessToken 갱신 성공')
+        return true
+      } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(no-unused-vars)